ipa_room_exploration/ros/src/room_exploration_evaluation.cpp

```diff
@@ -399,9 +399,6 @@ class ExplorationEvaluation
 				{
 					room_maps.push_back(room_map);
 					chosen_bb.push_back(bounding_boxes[room-1]);
-//					cv::rectangle(room_map, bounding_boxes[room-1], cv::Scalar(127), 2);
```

Comment:
Do NOT change anything in ipa_room_exploration/ros/src/room_exploration_evaluation.cpp - revert everything! This is a file for our evaluation of a paper, this HAS to be AS IS.